src/core/client/admin/components/UserRole/PromoteUserRoleMutation.ts

```diff
@@ -0,0 +1,45 @@
+import { graphql } from "react-relay";
+import { Environment } from "relay-runtime";
+
+import {
+  commitMutationPromiseNormalized,
+  createMutation,
+  MutationInput,
+} from "coral-framework/lib/relay";
+
+import { PromoteUserRoleMutation as MutationTypes } from "coral-admin/__generated__/PromoteUserRoleMutation.graphql";
+
+let clientMutationId = 0;
+
+const PromoteUserRoleMutation = createMutation(
+  "updateUserModerationScopes",
+  (environment: Environment, input: MutationInput<MutationTypes>) =>
+    commitMutationPromiseNormalized<MutationTypes>(environment, {
+      mutation: graphql`
+        mutation PromoteUserRoleMutation($input: PromoteUserRoleInput!) {
+          promoteUserRole(input: $input) {
+            user {
+              id
+              role
+              moderationScopes {
+                scoped
+                sites {
+                  id
+                  name
+                }
+              }
+            }
+            clientMutationId
+          }
+        }
+      `,
+      variables: {
+        input: {
+          ...input,
+          clientMutationId: (clientMutationId++).toString(),
+        },
+      },
+    })
+);
```

Comment:
This looks like it was copied over from the other PR!

Reply:
I branched off the other PR, but yes, you're very correct haha